test/Datadog.Trace.ClrProfiler.IntegrationTests/IIS/LoaderOptimizationStartup.cs

```diff
@@ -0,0 +1,58 @@
+#if NETFRAMEWORK
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.IIS
+{
+    public class LoaderOptimizationStartup
+    {
+        private const string Url = "http://localhost:8080";
+
+        public LoaderOptimizationStartup(ITestOutputHelper output)
+        {
+            this.Output = output;
+        }
+
+        private ITestOutputHelper Output { get; }
+
+        [Fact]
+        [Trait("RunOnWindows", "True")]
+        public void ApplicationDoesNotReturnErrors()
+        {
+            var intervalMilliseconds = 500;
+            var intervals = 5;
+            var serverReady = false;
+            var client = new HttpClient();
+
+            // wait for server to be ready to receive requests
+            while (intervals-- > 0)
+            {
+                try
+                {
+                    var serverReadyResponse = client.GetAsync(Url).GetAwaiter().GetResult();
```

Comment:
xUnit supports `async` test methods. Should we change this to use `await` instead of `GetAwaiter().GetResult()`?

Reply:
Addressed in 60ad318fb3fbe2d48b3477163178326edb021cc3